orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/tasks/rollingpush/WaitForNewUpInstancesLaunchTask.groovy

```diff
@@ -45,7 +45,6 @@ class WaitForNewUpInstancesLaunchTask implements RetryableTask {
 
     // similar check in `AbstractInstancesCheckTask`
     def response = oortService.getServerGroup(
-      stageData.application,
```

Comment:
This is backwards incompatible as it breaks the API contract with CloudDriver (unless you update CloudDriver, too).  In my opinion this should not have been merged as written.

Why not try the application specific endpoint after a 404?  The app specific endpoint is marked as deprecated (thank you) and can be logged out as such.  That would give folks some time to update CloudDriver (perhaps as part of a blessed release).

I will probably put up a PR for that if it seems reasonable.

Reply:
You should (ideally) be installing Spinnaker from a list of [verified versions](https://www.spinnaker.io/community/releases/versions/). We publish those to avoid having to add all the complexity of supporting multiple endpoints for several releases.

Reply:
We've been trying to stick with the verified versions, but due to various issues across Spinnaker services we inevitably need to update to newer versions while we keep other services set at the verified version.  I agree that ideally we could always stick with the verified versions, but as of yet that has not been the case.

Reply:
When issues come up can you file them in spinnaker/spinnaker/issues? This way we can help keep Spinnaker as up-to-date as possible by publishing verified cross-component releases.

Reply:
Sure.  In that same note - how do we prod you guys for a new verified release?  Currently 1.4.2 features a number of bugs around Orca pipeline templates and we can't make any progress until we update both Orca and CloudDriver, which we are hesitant to due because we continue to bump into problems (like this one) in our testing environment.

The fundamental problem I see with trying to bundle Spinnaker verified releases is that you're treating it like a monolithic release, whereas Spinnaker is really a set of microservices.  If semver would be taken a bit more seriously, problems like this wouldn't come about and we could treat Spinnaker like a set of microservices (with confidence) rather than waiting on the "verified release".

Reply:
@jonsie here's a good overview from @jtk54 about the release process for verified releases if you're interested: https://www.youtube.com/watch?v=aQ1lhi1l18s